modules/webbrowser/include/modules/webbrowser/properties/templatepropertywidgetcef.h

```diff
@@ -66,12 +66,24 @@ class TemplatePropertyWidgetCEF : public PropertyWidgetCEF {
         const std::string& requestString = request;
         auto j = json::parse(requestString);
         try {
-            T value = j.at("value").get<T>();
+            auto command = j.at("command").get<std::string>();
             auto p = static_cast<TemplateProperty<T>*>(getProperty());
-            p->setInitiatingWidget(this);
-            p->set(value);
-            p->clearInitiatingWidget();
-            callback->Success("");
+            if (command == "property.set") {
+                T value = j.at("value").get<T>();
+                
+                p->setInitiatingWidget(this);
+                p->set(value);
+                p->clearInitiatingWidget();
+                callback->Success("");
+            } else if (command == "property.get") {
+                if constexpr (std::is_same_v<T, bool>) {
```

Comment:
we have not switched to 17 yet...

Reply:
Knew this comment would come ;)
I don't think that too many people use this module and since we are switching soon anyway I will go ahead and merge it, ok?

Reply:
The module is not enabled by deafault so I think it is ok

Reply:
I was thinking we would make a clean c++14 release/tag first...

Reply:
Fixed